### DISCUSSION TITLE
How to get the old `--hold` behavior back?

### DISCUSSION THREAD

**Discussion by salmankhilji** [kovidgoyal/kitty#6587] [Q&A] [answered]
Starting with version 0.29, or specifically, with the following commit:

```
0fb1f17af Have --hold wait at a shell prompt 9 weeks ago
```

kitty now ends with at a shell prompt as opposed to asking the user to press a key to close the window.

The new behavior is breaking my workflow.  (I am running with `shell_integration disabled`.)

Is it possible to add an option to get the old behavior back?

**Comment by kovidgoyal** [comment]
I dont follow, how is it breaking your workflow? The only change is that
now instead of pressing enter/esc to exit you press ctrl+d. Also what
does shell_integration have to do with it? Are you saying it isnt
working without shell_integration?

**Reply by salmankhilji** [reply]
Its not really a "broken workflow" but old muscle memory.  I used to simply enter whereas now I must press ctrl+d to exit.

**Reply by kovidgoyal** [reply, marked as answer]
https://github.com/kovidgoyal/kitty/commit/37938573ca00eea97927b3e946a5272b0d5f2a3f
